src/App.vue

```diff
@@ -78,6 +84,25 @@ export default {
      */
     toggleFavourites() {
       this.showFavourites = !this.showFavourites;
+    },
+    /**
+     * Updates myFavourites array of beer ID's.
+     */
+    updateMyFavourites(id) {
```

Comment:
I feel like this one could be simplified. You are going through the array twice  (find and filter) which you could do once.

As `this.myFavourites` contains only id, you can use `indexOf` to get the index of the id in the array. If it exist, then you can use `splice` to remove the item. If it doesn't, you can push the id to the array. 

What you did might be better for complex list of objects but for such a simple array, it can be simplified.

Reply:
Updated function.